src/DIRAC/Resources/IdProvider/CheckInIdProvider.py

```diff
@@ -16,8 +16,13 @@ def getGroupScopes(self, group):
         """
         idPScope = getGroupOption(group, "IdPRole")
         if not idPScope:
-            idPScope = "eduperson_entitlement?value=urn:mace:egi.eu:group:{}:role={}#aai.egi.eu".format(
-                getVOForGroup(group),
-                group.split("_")[1],
-            )
+            vo = getVOForGroup(group)
+            if not vo:
+                return []
+
+            groupElements = group.split("_")
+            if len(groupElements) < 2:
+                return []
```

Comment:
The "DIRAC group" is something like "lhcb_user" or "dteam_prod" I guess...? But there's no rule (that I know) where this convention is enforced, or I am wrong? (this has of course nothing to do with this PR, just I happen to notice this only now)
In LHCb production we (historically) have for example a "diracAdmin" group.

Reply:
Indeed, I have no idea whether there is a rule for that.

Reply:
There is no rule for that. But this can be the moment to introduce it

Reply:
There surely shouldn't be any rule like that, that is way too constraining.

Reply:
Agreed